CRM/Core/Action.php

```diff
@@ -315,7 +315,7 @@ public static function formLink(
     $url = [];
 
     usort($seqLinks, static function ($a, $b) {
-      return (int) ((int) ($a['weight'] ?? 0) > (int) ($b['weight'] ?? 0));
+      return (int) ((int) ($a['weight']) > (int) ($b['weight']));
```

Comment:
@eileenmcnaughton What's wrong with having a default `weight` of `0`? That seems like a pretty standard default for `weight` and `priority` fields.

FWIW, I see a whole bunch of warnings on 5.66+master in `/civicrm/mailing/browse/scheduled?reset=1&scheduled=true` (*when there is  a scheduled mailing*)

Reply:
@totten yeah - we exposed them as notices cos it's not taking long having done this to ensure they are all set - these are passed out through hooks so we are going for consistency. It just takes a few minutes per page they show up on to fix

Reply:
Note the point of setting weights is to allow extensions to modify them (if you have ever seen the docs telling people how to re-order links you will understand why) - so getting them stable in a fairly short time frame will assist that  - I did fix the one you hit https://github.com/civicrm/civicrm-core/pull/27480